api-controllers/routes/admin.js

```diff
@@ -32,6 +32,12 @@ router.route('/backups')
 router.route('/backups/:backup_guid/delete')
   .post(controller.handler('deleteBackup'))
   .all(commonMiddleware.methodNotAllowed(['POST']));
+router.route('/config/create')
+  .post(controller.handler('createUpdateConfig'))
+  .all(commonMiddleware.methodNotAllowed(['POST']));
```

Comment:
once above line is changed, change here also to PUT.

Reply:
sure!